3-extra/1-count-words.js

```diff
@@ -25,10 +25,21 @@
 
```

Comment:
This implementation works - but is maybe a little more complex than it needs to be.
Can you think of how to simplify this? 🤔 Some ideas to to help find a simpler solution:
- Use the debugger in VsCode to help you step through the code and see what it is doing
- Think about whether you really need the `for` loop inside the function being passed to `forEach`